Ghidra/Features/Decompiler/src/main/doc/sleigh.xml

```diff
@@ -3525,6 +3525,7 @@ define space ram type=ram_space size=4 default;
 define space register type=register_space size=4;
 define register offset=0 size=4 [ r0 r1 r2 r3 r4 r5 r6 r7 ];
 define register offset=0x100 size=4 [ s0 s1 s2 s3 s4 s5 s6 s7 ];
+define register offset=0x180 size=4 [ statusreg ];
```

Comment:
I believe you meant to define contextreg here.  statusreg is already defined, and is used for something different than context.